policy/modules/services/stubby.te

```diff
@@ -0,0 +1,46 @@
+policy_module(stubby, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type stubby_t;
+type stubby_exec_t;
+init_daemon_domain(stubby_t, stubby_exec_t)
+
+type stubby_etc_t;
+files_config_file(stubby_etc_t)
+
+type stubby_unit_t;
+init_unit_file(stubby_unit_t)
+
+########################################
+#
+# Local policy
+#
+
+allow stubby_t self:tcp_socket create_stream_socket_perms;
+allow stubby_t self:udp_socket create_stream_socket_perms;
+
+read_files_pattern(stubby_t, stubby_etc_t, stubby_etc_t)
+# for /etc/trusted-key.key
+files_read_etc_files(stubby_t)
+miscfiles_read_generic_certs(stubby_t)
+miscfiles_read_localization(stubby_t)
+sysnet_read_config(stubby_t)
```

Comment:
Lines 27-31 should go after the corenet lines and also have additional spaces between the various module groups.